src/main/java/org/spongepowered/api/command/spec/CommandSpec.java

```diff
@@ -354,6 +354,15 @@ public boolean testPermission(CommandSource source) {
     }
 
     /**
+     * Gets the extended description used with this command if any is present
+     *
+     * @return the extended description.
+     */
+    public Optional<Text> getExtendedDescription(CommandSource source) {
```

Comment:
That would be cool ST, but sounds pretty intense refactor-wise; this PR was simply to be able to get access to the `extendedDescription` value of commands, not introduce this whole abstract layer of permission handling and stuff 😛 .